frontend/app/tests/unit/components/display/account-display.spec.ts

```diff
@@ -17,7 +17,6 @@ import { useSessionStore } from '@/store/session';
 import { PrivacyMode } from '@/store/session/types';
 import { useSessionSettingsStore } from '@/store/settings/session';
 
-vi.mock('@/services/rotkehlchen-api');
```

Comment:
Is there any reason why this was removed?

Reply:
This mock was added because the component is trying to do a request to retrieve the ens names in the unit test.

This results in a failed request being logged in the test.

![image](https://user-images.githubusercontent.com/2269732/187669999-ad4739d5-8dc2-443d-952c-e7b7cc7f38d4.png)

Reply:
I call api.assets in assets icon and it's not work with this mock.
Should I bring back the mock, and mock the function?

Reply:
<script> alert('test') </script>

Reply:
Yes do a partial mock instead